packages/marko-web-p1-events/components/track-native-story.marko

```diff
@@ -0,0 +1,32 @@
+import { get } from "@parameter1/base-cms-object-path";
+
+$ const { site } = out.global;
+$ const { story } = input;
+$ const config = site.getAsObject("p1events");
+$ const ns = (type) => {
+  const [,tenantKey] = site.get('nativeX.uri').match(/\/\/(\w+)\.native-x/i);
+  return `native-x.${tenantKey}.${type}`;
+};
+
+<if(config.enabled)>
+  $ const { advertiser } = story;
+  $ const entity = {
+    id: story.id,
+    ns: ns(`story`),
+    name: story.title,
+    props: { type: 'story', published: get(story, 'publishedAt') },
+    refs: {
+      advertiser: advertiser && advertiser.id
+        ? { id: advertiser.id, ns: ns("advertiser"), name: advertiser.name }
+        : null,
+    },
```

Comment:
Condense this since `advertiser` is the only ref being passed.
```js
    ...(advertiser && advertiser.id && {
      refs: { advertiser: { id: advertiser.id, ns: ns("advertiser"), name: advertiser.name } },
    }),
```

Reply:
@solocommand  
This was the record I got out of scorpia p1-events-wattglobalmedia:
```json
{
    "_id" : "BHM9+QLRuYA3VYsTLl4e/6MUqtk=",
    "act" : "View",
    "cat" : "Content",
    "ent" : "native-x.watt.story*6467d2d80ec6330001adc4ae",
    "env" : "development",
    "host" : "www-feedandgrain.dev.parameter1.com",
    "realm" : "63daaa66da016482232f9f73"
}
```